BUILD.md

```diff
@@ -39,7 +39,7 @@ Use this command for building and running the viewer:
 pixi run rerun
 ```
 
-This custom cargo command is enabled by an alias located in `.cargo/config.toml`.
+It is recommended you add `target/debug` to the start of your `PATH`. That way, all examples that call `spawn` will use your latest built `rerun` binary.
```

Comment:
Controversial, cf. Andreas' comments on slack (which I agree with—I won't do that myself). Also, I generally really dislike build instructions with intrusive stuff such as editing $PATH. We should strive to make that unnecessary.

Reply:
Maybe we can modify `spawn` to look for `target/debug/rerun` in debug builds 🤔

Reply:
* Done: https://github.com/rerun-io/rerun/pull/7437